gluetool/proxy.py

```diff
@@ -33,7 +35,7 @@ def __delattr__(self, name):
     def __setattr__(self, name, value):
         setattr(object.__getattribute__(self, "_obj"), name, value)
 
-    def __nonzero__(self):
+    def __boolo__(self):
```

Comment:
We need both - Python 2 will try to call ``__nonzero__``.

```python
def __bool__
  return bool(object.__getattribute__(self, "_obj"))

if six.PY2:
  __nonzero__ = __bool__
```

Also, typo: `s/boolo/boo/`

Reply:
thanks, will be in next patch